src/core/row.cpp

```diff
@@ -26,6 +26,31 @@ row::~row()
     clean_up();
 }
 
+#ifdef SOCI_HAVE_CXX11
```

Comment:
_If_ we keep this code at all, we don't need this test any longer in master.

Reply:
Oh, I intended to remove this as I did in the header file.